Dockerfile.ci

```diff
@@ -78,7 +78,6 @@ RUN apt-get update \
            libsasl2-dev \
            libsasl2-modules \
            libssl-dev \
-           libenchant-dev \
```

Comment:
I believe this one is not needed any more. Let's see if all tests pass (this is a spellcheck library with some rather obscure use).

Reply:
Ah no... It's used by sphinx :(